core/access_api.php

```diff
@@ -333,6 +333,69 @@ function access_has_project_level( $p_access_level, $p_project_id = null, $p_use
 	return access_compare_level( $t_access_level, $p_access_level );
 }
 
+
+/**
+ * Check the current user's access against the given value, in each of the provided projects,
+ * and return true if the user's access is equal to or higher in any of the projects, false otherwise.
+ * @param integer|array|string  $p_access_level Parameter representing access level threshold, may be:
+ * - integer: for a simple threshold
+ * - array: for an array threshold
+ * - string: for a threshold option which will be evaluated for each project context
+ * @param array                 $p_project_ids  Array of project ids to check access against, default to null to use all user accesible projects
+ * @param integer|null          $p_user_id      Integer representing user id, defaults to null to use current user.
+ * @return boolean whether user has access level specified
```

Comment:
Wrap long lines.
Bullets for $p_access_level should be indented below the param's description